tool/tool.go

```diff
@@ -0,0 +1,105 @@
+package tool
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/urfave/cli"
+
+	"github.com/tidepool-org/platform/application"
+	"github.com/tidepool-org/platform/log"
+)
+
+const (
+	HelpFlag    = "help"
+	VersionFlag = "version"
+	VerboseFlag = "verbose"
+)
+
+type Tool struct {
```

Comment:
If its always a command line tool do you think the naming should reflect that?

Reply:
It doesn't have to be. I expect that certain tools will be invoked from the admin site, once that gets implemented, via exec calls.